components/server/src/config/config-inferrer.ts

```diff
@@ -83,6 +83,16 @@ export class ConfigInferrer {
             this.addExtension(ctx, "vscjava.vscode-java-debug");
             return;
         }
+        // Gradle Kotlin DSL
+        if (await ctx.exists("build.gradle.kts")) {
+            let cmd = "gradle";
+            if (await ctx.exists("gradlew")) {
+                cmd = "./gradlew";
+            }
+            this.addCommand(ctx.config, cmd + " build", "init");
+            this.addExtension(ctx, "fwcd.kotlin");
```

Comment:
💭 **Not related to this change**: But, these extensions don't seem to be inflated in `.gitpod.yml` files (even for other language inferrers).

Reply:
@Siddhant-K-code could you share what you mean by "inflated"? Are they not propagating to the resulting ymls correctly?

Reply:
> Are they not propagating to the resulting ymls correctly?

yes